apps/sms/services/js/threads.js

```diff
@@ -164,5 +164,18 @@
     }
   };
 
+  Threads.Messages = {
+    get(id) {
+      id = +id;
+      var threadId = messageMap.get(id);
+
+      if (Threads.has(threadId)) {
+        return Threads.get(threadId).messages.get(id);
+      }
+
+      return null;
+    }
```

Comment:
> Sorry if this messageMap is not clear because it's create by me :p.

Ha-ha, no worries! I just wasn't aware of it, so was a bit confused by name :)